disco_aws_automation/disco_metanetwork.py

```diff
@@ -236,3 +251,28 @@ def add_sg_rule(self, protocol, ports, sg_source=None, cidr_source=None):
         sg_args["to_port"] = ports[1]
         logging.debug("Adding sg_rule: %s", sg_args)
         self.vpc.vpc.connection.authorize_security_group(**sg_args)
+
+    def ip_by_offset(self, offset):
+        """
+        Pass in +10 and get 10th ip of subnet range
+        Pass in -2 and get 2nd to last ip of subnet
+
+        Returns IpAddress object, usually you'll want
+        to cast this to str.
+        """
+
+        try:
```

Comment:
Not sure what this `try` - `except` does. Seems like trying if conversion to *int* works, and throwing an error otherwise.  How is it testing if the offset is trying to set an IP address outside of metanetwork range